src/odemis/driver/xt_client.py

```diff
@@ -2676,15 +2709,14 @@ def __init__(self, name, role, parent, hfw_nomag, channel="electron1", **kwargs)
         # Instantiate the super scanner class with the update thread
         super(MultiBeamScanner, self).__init__(name, role, parent, hfw_nomag, **kwargs)
 
-    @isasync
     def applyAutoStigmator(self):
         """
-        Wrapper for autostigmation flash function, non-blocking.
+        Wrapper for autostigmation function, blocking.
```

Comment:
If it returns a future, it's *not* blocking... And as it's supposed to take ~90s it sounds useful that it's not blocking!